### DISCUSSION TITLE
interview/javascript/1.%E6%95%B0%E6%8D%AE%E7%B1%BB%E5%9E%8B1-%E6%95%B0%E6%8D%AE%E7%B1%BB%E5%9E%8B%E4%B8%8E%E5%AD%98%E5%82%A8

### DISCUSSION THREAD

**Discussion by giscus** [dllcnx/dllcnx.github.io#45] [Announcements]
# interview/javascript/1.%E6%95%B0%E6%8D%AE%E7%B1%BB%E5%9E%8B1-%E6%95%B0%E6%8D%AE%E7%B1%BB%E5%9E%8B%E4%B8%8E%E5%AD%98%E5%82%A8

根据最新的 ECMAScript 标准定义了 8 种数据类型，可以分为 2 大类： 基本数据类型:string,`number,null,Boolean,undefined,symbol,BigInt; 引用数据类型: Object[Array,Function ]; 区别: 基本数据类型保存在栈里面,可以直接访问他的值,引用数据类型保存在堆里面,栈里...

https://dllcnx.com:10000/interview/javascript/1.%E6%95%B0%E6%8D%AE%E7%B1%BB%E5%9E%8B1-%E6%95%B0%E6%8D%AE%E7%B1%BB%E5%9E%8B%E4%B8%8E%E5%AD%98%E5%82%A8.html

**Comment by dllcnx** [comment]
测试一下
